components/EditProfilePage/FollowingTab.tsx

```diff
@@ -99,11 +171,12 @@ export function FollowingTab({ className }: Props) {
         <div className={`mx-4 mt-3 d-flex flex-column gap-3`}>
           <Stack>
             <h2>{t("follow.bills")}</h2>
-            {billsFollowing.map((element: any) => (
+            {billsFollowing.map((element: string, index: number) => (
               <FollowedItem
-                key={element.billId}
+                key={index}
```

Comment:
I agree that this probably would be the best way to go. The reason I made this change is it looks like the billsFollowing and orgsFollowing arrays were converted from objects to strings several months back, but the FollowedItem code wasnt changed along with it. That being said, it could be worth changing them back to objects, if it means having better consistency